lib/elastic_record/relation/batches.rb

```diff
@@ -29,11 +29,25 @@ def build_scroll_enumerator(options)
         elastic_index.build_scroll_enumerator(search: as_elastic, **options)
       end
 
+      def build_search_after(options)
+        elastic_index.build_search_after(search: as_elastic, **options)
+      end
+
       def reindex
         relation.find_in_batches do |batch|
           elastic_index.bulk_add(batch)
         end
       end
+
+      private
+
+        def build_paginator(options)
+          if options.delete(:paginator) == :search_after
+            build_search_after(options)
+          else
+            build_scroll_enumerator(options)
+          end
```

Comment:
I think this is a good approach while we test `search_after` in production environments.